conf/application.conf

```diff
@@ -75,6 +75,12 @@ microservice {
   }
 }
 
+schedules {
+ eu-vat-rates-worker {
+  interval = 1 day
```

Comment:
can we have `interval-days`? To reduce human error? I know the value is `1 day` indicating the frequency but might be blind to an override config.

Reply:
The type is `FiniteDuration`. An override config would also have to include the unit, whether it's `1 day` or `24 hours` etc.

Reply:
Yes that is true but if an application operates on interval of 1 day and suddenly changes to `10 seconds` or `48 hours` this will have an unpredictable impact on the infrastructure which the application was not designed to cope with. It helps us know the applications limitations/expectations if we got a clue of the expected unit from the property name in the config.

Reply:
It can be days, hours, seconds, milliseconds, there isn't an expectation from the application level, if performance is affected, that's the job of the performance test to hand. I can't arbitrarily say that it's only meant to be days. That's just what I've picked as sensible for now and may change in the future